app/models/manageiq/providers/foreman/configuration_manager.rb

```diff
@@ -31,11 +35,29 @@ def self.description
     @description ||= "Foreman Configuration".freeze
   end
 
+  def self.create_from_params(params, endpoints, authentications)
+    parama[:name] = "#{params[:name]} Configuration Manager" unless params[:name].end_with?(" Configuration Manager")
```

Comment:
This is just to keep this consistent with what `Provider#ensure_managers` is doing: https://github.com/ManageIQ/manageiq-providers-foreman/blob/c12df78f77b7f13563b4150df41145bd8ba5b3ed/app/models/manageiq/providers/foreman/provider.rb#L127-L131